README.md

```diff
@@ -1,67 +1,79 @@
-# cf-cdn-service-broker
+# Cloud Foundry CDN Service Broker
 
-A Cloud Foundry service broker for CloudFront and Let's Encrypt
+A [Cloud Foundry](https://www.cloudfoundry.org/) [service broker](http://docs.cloudfoundry.org/services/) for [CloudFront](https://aws.amazon.com/cloudfront/) and [Let's Encrypt](https://letsencrypt.org/).
 
 ## Deployment
 
-* Create services
+1. Clone this repository, and `cd` into it
+1. Target the space you want to deploy the broker to
 
-    Note: the following credentials must be available in the `cdn-creds` service or in top-level environment variables:
+    ```bash
+    $ cf target -o <org> -s <space>
+    ```
 
-    * PORT
-    * BROKER_USER
-    * BROKER_PASS
-    * DATABASE_URL
-    * EMAIL
-    * ACME_URL
-    * BUCKET
-    * AWS_ACCESS_KEY_ID
-    * AWS_SECRET_ACCESS_KEY
-    * AWS_DEFAULT_REGION
+1. Create a [user-provided service](http://docs.cloudfoundry.org/devguide/services/user-provided.html) with the configuration (you will be prompted for the values of each)
 
-    ```
+    ```bash
     $ cf create-service rds micro-psql cdn-rds
-    $ cf create-user-provided-service cdn-creds -p '{"AWS_ACCESS_KEY_ID": "[key-id]", ...}'
+    $ cf create-user-provided-service cdn-creds -p 'BROKER_USER,BROKER_PASS,DATABASE_URL,EMAIL,ACME_URL,BUCKET,AWS_ACCESS_KEY_ID,AWS_SECRET_ACCESS_KEY,AWS_DEFAULT_REGION'
     ```
 
-* Deploy application
+1. Deploy the broker as an application
 
-    ```
+    ```bash
     $ cf push
     ```
 
-* Add to Cloud Foundry
+1. [Register the broker](http://docs.cloudfoundry.org/services/managing-service-brokers.html#register-broker)
 
-    ```
+    ```bash
     $ cf create-service-broker cdn-route [username] [password] [app-url] --space-scoped
     ```
 
 ## Usage
 
-* Create service
+1. Target the space your application is running in
 
+    ```bash
+    $ cf target -o <org> -s <space>
     ```
+
+1. Create service instance
+
+    ```bash
     $ cf create-service cdn-route cdn-route my-cdn-route \
         -c '{"domain": "my.domain.gov", "origin": "my-app.apps.cloud.gov"}'
 
     Create in progress. Use 'cf services' or 'cf service my-cdn-route' to check operation status.
     ```
 
-* Get CNAME instructions
+1. Get CNAME instructions
 
-    ```
+    ```bash
     $ cf service my-cdn-route
 
     Last Operation
     Status: create in progress
     Message: Provisioning in progress; CNAME domain "my.domain.gov" to "d3kajwa62y9xrp.cloudfront.net."
     ```
 
-* Update CNAME
+1. Create/update your CNAME in your DNS configuration
+1. Create a [private domain](http://docs.cloudfoundry.org/devguide/deploy-apps/routes-domains.html#private-domains) in your Cloud Foundry organization
```

Comment:
So what exactly does this do? I haven't been using private domains as I've been testing this, and cloud foundry doesn't have to know anything about the external domain.